charts/kong/README.md

```diff
@@ -680,6 +681,32 @@ For a complete list of all configuration values you can set in the
 | extraConfigMaps                    | ConfigMaps to add to mounted volumes                                                  | `[]`                |
 | extraSecrets                       | Secrets to add to mounted volumes                                                     | `[]`                |
 
+**Note:** If you are using `deployment.hostNetwork` to bind to lower ports ( < 1024), which may be the desired option (ports 80 and 433), you also 
+need to tweak the `containerSecurityContext` configuration as in the example:
+
+```yaml
+deployment:
+  daemonset: true # run as daemonset
+  hostNetwork: true # enable hostNetwork
+nodeSelector: # limit the nodes bases on a label
+  ingress: "true"
+proxy:
+  type: NodePort # maybe you don't need or want a LoadBalancer
+  http:
+    containerPort: 80 # containerPor and hostPort neest to be the same
+    nodePort: 32080
+    hostPort: 80
+  tls:
+    containerPort: 443
+    nodePort: 32443
+    hostPort: 443
+containerSecurityContext: # run as root to bind to lower ports
+  capabilities:
+    add: [NET_BIND_SERVICE]
+  runAsGroup: 0
+  runAsNonRoot: false
+  runAsUser: 0
```

Comment:
Minor change to make this focus on the container security context changes only:
```suggestion
containerSecurityContext: # run as root to bind to lower ports
  capabilities:
    add: [NET_BIND_SERVICE]
  runAsGroup: 0
  runAsNonRoot: false
  runAsUser: 0
```

Reply:
It's done